Automattic/jetpack
# Add jetpack-ip package functionality

#### Description
Adds methods to the primary `Utils` class in the `jetpack-ip` package for IP address handling.

## Proposed changes:

* Duplicates all Brute force protection IP address shared functions to the `jetpack-ip` package `Utils` class.
* Duplicates all Brute force protection IP address tests to `jetpack-ip` package.

### Other information:

## Jetpack product discussion


- #28765 

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Proofread
* Checkout this branch
* Run `composer phpunit` within the `jetpack-ip` package directory
* Verify that tests all pass
* Start up your Jurassic Tube
* Enable Jetpack and connect your user
* Test Brute force protection functionality to ensure that no issues are introduced
